fiftyone/core/collections.py

```diff
@@ -3339,6 +3339,21 @@ def compute_metadata(
             progress=progress,
         )
 
+    def compute_annotation_schema(self, field_name):
+        """Compute the annotation schema for a field
+
+        Args:
+            field_name: the field to process
```

Comment:
nit: for consistency with other methods, I think we should explicitly document when a field name can be embedded:

```py
"""
            field_name: a field name or ``embedded.field.name`` to process
"""
```